<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>running_counts_widget</class>
 <widget class="QWidget" name="running_counts_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="running_counts_controls">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>162</width>
     <height>311</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QDoubleSpinBox" name="running_counts_dwell_time_spin_box">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>40</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>0.000200000000000</double>
    </property>
    <property name="maximum">
     <double>10000.000000000000000</double>
    </property>
    <property name="value">
     <double>5.000000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="running_counts_number_of_points">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
    <property name="value">
     <number>200</number>
    </property>
    <property name="displayIntegerBase">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="running_counts_push_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Run Counts</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Delay(s)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_running_counts_push_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Counts</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of Points</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="running_counts_delay_spin_box">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>70</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>0.050000000000000</double>
    </property>
    <property name="maximum">
     <double>10000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.050000000000000</double>
    </property>
    <property name="value">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Dwell Time (ms)</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="running_counts_plot_frame">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>531</width>
     <height>261</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="running_counts_toolbar_frame">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>531</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
